moduleframework/helpers/rpm_helper.py

```diff
@@ -46,14 +47,19 @@ def __init__(self):
         self.bootstrappackages = []
         self.repos = []
 
-    def getURL(self):
+    @property
+    def URL(self):
         """
         Return semicolon separated string of repositories what will be used, could be simialr to URL param,
          it contains also dependent repositories from PDC
 
         :return: str
         """
-        return ";".join(self.repos)
+        return ';'.join(self.repos)
+
+    @URL.setter
```

Comment:
this URL, should be just readonly, it should not be  allowed SET URL

Reply:
@jscotka And how would you like to setup? in __init__ function?